Makefile

```diff
@@ -289,20 +289,23 @@ public/js/index.js: $(JAVASCRIPTS)
 	cat $< >| $@
 
 .PHONY: stylesheets-check
-stylesheets-check: stylesheets
+stylesheets-check: generate-stylesheets
 	@diff=$$(git diff public/css/index.css); \
 	if [ -n "$$diff" ]; then \
-		echo "Please run 'make stylesheets' and commit the result:"; \
+		echo "Please run 'make generate-stylesheets' and commit the result:"; \
 		echo "$${diff}"; \
 		exit 1; \
 	fi;
 
-.PHONY: stylesheets
-stylesheets:
+.PHONY: generate-stylesheets
+generate-stylesheets:
+	node_modules/.bin/lessc --no-ie-compat public/less/index.less public/css/index.css
+
+.PHONY: minify
```

Comment:
Maybe rename this build step to `minify-stylesheets` ?

Reply:
@daviian this step will later be used also to minify js files etc, so can be left as is

Reply:
@lafriks That's exact reason for that name.